prow/plugins/updateconfig/updateconfig_test.go

```diff
@@ -298,123 +453,10 @@ func TestUpdateConfig(t *testing.T) {
 		for _, configName := range tc.configUpdates {
```

Comment:
nit: Can we use `updatedConfigMaps` here and get rid of the `configUpdates` field?

It would also be good to check that we didn't update any config maps that we didn't expect to update.

Reply:
done

I think the foo.txt test case covers that we didn't update unaffected configmaps?

Reply:
I think that just tests that we didn't update any unaffected configmap keys, not the configmaps themselves.

Reply:
https://github.com/krzyzacy/test-infra/blob/c95d6041416e86559b635a6960c2e84d89ca2b11/prow/plugins/updateconfig/updateconfig_test.go#L293-L329 should be this test case is doing?

Reply:
There is only one configmap in that example (multikey-config). We check that only the `foo.yaml` **key** is updated, but we don't have any checks to validate that other preexisting configmaps (not configmap keys) are not updated.

Reply:
Also it doesn't look like the `configUpdates` field was actually removed.

Reply:
done I think?